source/channels/join-leave-channels.rst

```diff
@@ -11,27 +11,45 @@ To join a private channel, you need to be added to the channel by a member of th
 
 To join a public channel:
 
-1. Select the **+** symbol at the top of the channel sidebar, then select **Browse Channels**. 
+.. tabs::
 
-  .. image:: ../images/browse-channels.png
-    :alt: You can browse available channels to join using the + option at the top of the channel sidebar.
+  .. tab:: Web browser/Desktop app
 
-2. Select **Join** next to the public channel you want to join.
+    1. Select the **+** symbol at the top of the channel sidebar, then select **Browse Channels**. 
 
-  .. image:: ../images/join-channels.png
-    :alt: When browsing available channels, select the join option next to any channel to become a member.
+      .. image:: ../images/browse-channels.png
+        :alt: You can browse available channels to join using the + option at the top of the channel sidebar.
 
-.. tip::
-  
-  When you join channels, depending on the `channel actions configured </channels/create-channels.html>`__, you may see a welcome message, and channels may be added to `a category in your channel sidebar </channels/customize-your-channel-sidebar.html>`__ automatically. 
-  Access **Channel Actions** from the channel name drop-down menu in the center pane to see what automatic actions have been configured.
+    2. Select **Join** next to the public channel you want to join.
+
+      .. image:: ../images/join-channels.png
+        :alt: When browsing available channels, select the join option next to any channel to become a member.
+
+    .. tip::
```

Comment:
Please move this note to follow the tabbed content for joining a channel since it applies to both tabs overall. This would make the tab appear more consistently too with the leave channel content that applies in all cases.